spec/forms/password_form_spec.rb

```diff
@@ -4,10 +4,6 @@
   subject { PasswordForm.new(build_stubbed(:user)) }
 
   it do
-    is_expected.to validate_confirmation_of(:password)
-  end
```

Comment:
Because we no longer ask the user to type in their password twice. This refers to the additional password confirmation field we used to have and that we removed a while ago.